### PR TITLE
README.md Draft Ready for Review

### PR DESCRIPTION
**Pull request by danzsmith53**
@ashahba Please review my proposed changes to the original README.md file. Dan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
 
-#jupyter
+#spark-tk-jupyter
 
-- This is the home of docker image for Jupyter on TAP. 
+- This is the home of docker image for Jupyter for the TAP Analytics Toolkit (TAP).
```
Comment (ashahba) [inline README.md]: This is the home of Jupyter docker for the TAP Analytics Toolkit (TAP).

File: README.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+This is the initial release of the `spark-tk-jupyter` repo.
+
+##Known issues
+None.
+
 ## Building the image
-- Pull all the submodules: git submodule update --init --recursive
-- sudo docker build .
+- Pull all the submodules: `git submodule update --init --recursive`
+- `sudo docker build`
```
Comment (ashahba) [inline README.md]: `sudo docker build -f Dockerfile`

File: README.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
 - Or if you are behind a proxy use this:
-- sudo docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$http_proxy --build-arg NO_PROXY=$no_proxy --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy .
+- `sudo docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$http_proxy --build-arg NO_PROXY=$no_proxy --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy`
```
Comment (ashahba) [inline README.md]: `sudo docker build --build-arg HTTP_PROXY=$http_proxy --build-arg HTTPS_PROXY=$http_proxy --build-arg NO_PROXY=$no_proxy --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_proxy --build-arg no_proxy=$no_proxy -f Dockerfile`

**Comment by danzsmith53** [comment]
Ebi, I think it is ready for a merge. I do not have write access.

**Review by ashahba — APPROVED** [review]

**Pull request merged**